src/riscv/lib/src/state_backend.rs

```diff
@@ -495,55 +493,23 @@ pub(crate) mod test_helpers {
         ///
         /// Used for testing.
         pub trait TestBackendFactory = ManagerReadWrite
-            + ManagerSerialise
-            + ManagerDeserialise
             + ManagerClone
             + ManagerAlloc;
     }
-
-    /// Copy the allocated space by serialising and deserialising it.
-    pub fn copy_via_serde<L, N, M>(refs: &AllocatedOf<L, N>) -> AllocatedOf<L, M>
-    where
-        L: Layout,
-        N: ManagerSerialise,
-        AllocatedOf<L, N>: serde::Serialize,
-        M: ManagerDeserialise,
-        AllocatedOf<L, M>: serde::de::DeserializeOwned,
-    {
-        let data = crate::storage::binary::serialise(refs).unwrap();
-        crate::storage::binary::deserialise(&data).unwrap()
-    }
-
-    /// Assert that two values are different. If they differ, offer a command to run that shows the
-    /// structural differences between the values.
-    pub fn assert_eq_struct<T>(lhs: &T, rhs: &T)
-    where
-        T: serde::Serialize + PartialEq,
-    {
-        if lhs != rhs {
-            let (file_lhs, path_lhs) = tempfile::NamedTempFile::new().unwrap().keep().unwrap();
-            serde_json::to_writer_pretty(file_lhs, lhs).unwrap();
-            eprintln!("Lhs is located at {}", path_lhs.display());
-
-            let (file_rhs, path_rhs) = tempfile::NamedTempFile::new().unwrap().keep().unwrap();
-            serde_json::to_writer_pretty(file_rhs, rhs).unwrap();
-            eprintln!("Rhs is located at {}", path_rhs.display());
-
-            eprintln!("Run the following to diff them:");
-            eprintln!("jd {} {}", path_lhs.display(), path_rhs.display());
-
-            panic!("Assertion failed: values are different");
-        }
-    }
 }
 
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::backend_test;
     use crate::state::NewState;
 
-    backend_test!(test_example, F, {
+    #[test]
+    fn test_example_owned() {
+        use crate::state_backend::Cell;
+        use crate::state_backend::Cells;
+        use crate::state_backend::FnManagerIdent;
+        use crate::state_backend::owned_backend::Owned;
```

Comment:
Nit: feel free to resolve

```suggestion
    use crate::state::NewState;
    use crate::state_backend::Cell;
    use crate::state_backend::Cells;
    use crate::state_backend::FnManagerIdent;
    use crate::state_backend::owned_backend::Owned;

    #[test]
    fn test_example_owned() {
```